src/derivatives/atlas.md

```diff
@@ -8,45 +8,42 @@ segmentations, and other knowledge annotations such as landmarks defined with
 The following entities MAY be employed to specify template- and atlas-derived results:
 
 -    [`tpl-<label>`](../glossary.md#template-entities) is REQUIRED to specify derivatives defining
-     a [template](../common-principles.md).
+     a [template](../common-principles.md).  MUST NOT be used alongside `atlas-<label>`
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to specify derivatives defining an [atlas](../common-principles.md).  MUST NOT be used alongside `template-<label>`
 -    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
      respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
-     BIDS-Derivatives specifications.
+     BIDS-Derivatives specifications.  It is also REQUIRED when [`tpl-<label>`](../glossary.md#template-entities) is used,
+     and the template is NOT serving as the authoritative definition of a space.
 -    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
-     respect to different cohort instances of a single [space (coordinate system)](../appendices/coordinate-systems.md).
+     respect to different cohort instances.
      Please note that [`cohort-<label>`](../glossary.md#cohort-entities) MUST NOT be used if neither
-     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities) are used.
--    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
-     or derived from the atlas identified by the entity's label.
+     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities)
+     nor [`atlas-<labl>`](../glossary.md#atlas-entities) are used.
 -    [`seg-<label>`](../glossary.md#segmentation-entities) is REQUIRED when a single atlas has several different
      realizations (for instance, segmentations and parcellations created with different criteria) that
      need disambiguation.
 -    [`scale-<label>`](../glossary.md#scale-entities) is REQUIRED to disambiguate different atlas 'scales',
      when the atlas has more than one 'brain unit' resolutions, typically relating to the area covered
      by regions.
 
-The general filename pattern for subject derivatives with templates and atlases in their provenance
+The general filename pattern for subject-level derivatives with templates and atlases 
 follows the general BIDS-Derivatives pattern:
 
 ```Text
 <pipeline_name>/
     sub-<label>/
         <datatype>/
-            <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+            <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][_template-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Comment:
a template entity should not be introduced, but rather use the tpl entity?

Reply:
Yes, good catch.  I've reworked this so that it uses the `tpl` entity, and to clarify that one can either use the tpl entity, atlas entity, but not both.